pkgs/applications/editors/poke/default.nix

```diff
@@ -0,0 +1,85 @@
+{ lib
+, stdenv
+, fetchurl
+
+# Required build tools
+, bison
+, flex
```

Comment:
Nope, thanks :smile:! When I originally packaged it, they were required, but maybe it was just because I was building from the git sources, and now I'm using a release tarball: https://github.com/MetaDark/nur-packages/commit/3a80ae1f3e21ae2c6c980719a1f83a5dd2d3953a#diff-4e080b29ba712d25f8de1f6de6e7f91f88562eea78785e74b805e0ecc0d69a58